assets/js/components/OptIn.js

```diff
@@ -49,6 +49,7 @@ export default function OptIn( {
 	name = 'optIn',
 	className,
 	trackEventCategory,
+	trackEventAction = 'tracking_optin',
```

Comment:
Adding a `trackEventAction` prop is a lighter touch than adding a `gaTrackingEventArgs` prop as the IB suggested. If we add `gaTrackingEventArgs` we'll need to update the `Tracking` component as well.